src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.js

```diff
@@ -457,7 +446,9 @@ function ComposerWithSuggestions({
             }
 
             focus();
-            replaceSelectionWithText(e.key, false);
+            if (e.key.length === 1) {
```

Comment:
Moving this condition could potencially change the behaviour - regardless of `e.key.length` value we will focus the input. Before we would return early from the the function without trigerring `focus()`

Reply:
Wasn't the main part of the issue to not focus on modifier keys? Copying from the expected behaviour: `The Emoji search field is not focused when clicking on Modifier keys`

Reply:
D we still need this condition @dukenv0307 ? Beforehand in `shouldAutoFocusOnKeyPress` we are checking `whether e.key.length >1`